allennlp/tests/models/event2mind_test.py

```diff
@@ -55,22 +61,26 @@ def test_beam_search_orders_results(self):
     def test_beam_search_matches_greedy(self):
         model = self.trained_model
         state = model._states["xintent"]
+        beam_search = BeamSearch(model._end_index,
+                                 max_steps=model._max_decoding_steps,
+                                 beam_size=1)
+
+        final_encoder_output = self.get_sample_encoded_output()
+        batch_size = final_encoder_output.size()[0]
+        start_predictions = final_encoder_output.new_full(
+                (batch_size,), fill_value=model._start_index, dtype=torch.long)
+        start_state = {"decoder_hidden": final_encoder_output}
+
```

Comment:
It's been retrained now

Reply:
Thank you. The validation during training uses beam search. The stopping condition was actually based on a metric computed from a beam, so there was some chance running this would help catch breakage. (Incidentally, we're planning on automating this sort of thing: https://github.com/allenai/allennlp/issues/1813)